RSA_Algorithm/RSA_Algorithm.go

```diff
@@ -0,0 +1,82 @@
+// Program in Go language to implement RSA algorithm
+
+package main
+
+import (
+    "fmt"
+)
+
+//Function to calculate Greatest Common Divisor based on Euclid's Method
+func egcd(a int, b int) int {1
```

Comment:
what is the purpose of 1 here?

Reply:
It must have been a typo. I will rectify it.